plottable.js

```diff
@@ -3542,13 +3431,7 @@ var Plottable;
     })(Plottable.Component);
     Plottable.ComponentContainer = ComponentContainer;
 })(Plottable || (Plottable = {}));
-
 ///<reference path="../reference.ts" />
-var __extends = (this && this.__extends) || function (d, b) {
-    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
-    function __() { this.constructor = d; }
-    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
-};
```

Comment:
Could it be the reason why these lines are disappearing is that, previously, by compiling file-by-file, the definition of ```__extends``` was part of every file. Hence it appeared multiple times when plottable.js was concatenated. Now it is included by the compiler only once per closure.